assets/src/edit-story/components/library/panes/media/media3p/media3pPane.js

```diff
@@ -51,6 +51,12 @@ const ProviderTabSection = styled.div`
   padding: 0 24px;
 `;
 
+const MediaDisplayName = styled.div`
```

Comment:
< MediaSubheading >?

instead of media name maybe you should call it "media subheading" here in the comment and every instance of "MediaName".

MediaDisplayName is a bit confusing to me since it sounds like it is the name of each picture or the name of the media provider (i.e. unsplash).